core/src/main/java/io/questdb/client/Sender.java

```diff
@@ -338,6 +338,7 @@ enum TlsValidationMode {
     final class LineSenderBuilder {
         private static final int AUTO_FLUSH_DISABLED = 0;
         private static final int DEFAULT_AUTO_FLUSH_ROWS = 600;
+        private static final long DEFAULT_MIN_REQUEST_THROUGHPUT = 100_000; // 100KB/s, keep in sync with the contract of the configuration method
```

Comment:
`100 * 1024`?

Reply:
Not that anyone would be able to spot the difference, in all fairness :-)

Reply:
yes, this is a good idea.